src/app/learn/get-started/advanced/sticky-settings/overview/index.html

```diff
@@ -0,0 +1,84 @@
+<stache
+  pageTitle="Overview"
+  showTableofContents="true"
+  navOrder="10">
+
+  <stache-page-summary>
+    <p>
+      Sticky settings enable you to save user settings across sessions in flyouts, grids, and tile dashboards. Sticky settings preserve the <a stacheRouterLink="/learn/get-started/advanced/sticky-settings/integrations/skyux-components/flyouts">width of flyouts</a>, the <a stacheRouterLink="/learn/get-started/advanced/sticky-settings/integrations/skyux-components/grids">column order and the sorting of columns in grids</a>, and the <a stacheRouterLink="/learn/get-started/advanced/sticky-settings/integrations/skyux-components/tiles">layout and collapsed state of tiles in tile dashboards</a>.
+    </p>
+  </stache-page-summary>
+
+  <sky-alert>
+    For Blackbaud developers, the sticky settings functionality is available first-class through the UI Config Service that is for internal Blackbaud use only. Blackbaud developers can learn how to implement sticky settings on <a href="https://docs.blackbaud.com/ui-config-service-docs/">the UI Config Service website</a>.
+  </sky-alert>
+  <p>
+    To implement sticky settings, you must provide a class for <sky-code>SkyUIConfigService</sky-code> in your SPA's <sky-code>app-extras.module.ts</sky-code> file.
+  </p>
+  <sky-code-block languageType="typescript">
+    import {
+      AppUIConfigService
```

Comment:
Let's rename this class to 'AppUIConfigService`.

Reply:
I removed "Sky" in line 31 as discussed offline.